### ISSUE TITLE
Make `bento check` and `bento check .` behaviour the same

### ISSUE THREAD

**Issue by dlukeomalley** [returntocorp/bento#241]
## Overview
Both `bento check` and `bento check .` should return the same results and they do (for me 0 findings), but they behave differently.

`bento check` without any paths notifies me that there isn't anything to check when no files have changed in git, and takes no-time:

```
⚠ Nothing to check. By default, Bento only analyzes tracked files with diffs.
```

`bento check .` with a path does not notify me of this and takes much longer to run.

## Current Behavior

```bash
$ bento check
Running Bento checks on changes since last commit...

⚠ Nothing to check. By default, Bento only analyzes tracked files with diffs.
◦ To check a specific path, run $ bento check PATH

✔ 0 findings in 0.00 s
◦ To show all findings in the project, run $ bento check --comparison=archive .
```

```bash
$ bento check .
Running Bento checks on changes since last commit...

bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:20, 🍱 Done      ^C
flake8․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:20, 🍱 Done
hadolint․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■□□□□□□□□| 00:20, 🍤 Running
r2c.click․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:20, 🍱 Done
r2c.flask․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:20, 🍱 Done
r2c.requests․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:20, 🍱 Done
```

## Steps to Reproduce

1. `bento check`
2. `bento check .`

## Additional Context
Bento version (run `bento --version`): 0.8.0b8

**Comment by dlukeomalley** [comment]
@DrewDennison @brendongo I think we should make this release blocking. It's caused confusion for 3 distinct users during the pre-release.

**Comment by pabloest** [comment]
I was confused by this, and the confusion was compounded by `bento check --comparison=archive .` taking 1.23s compared to `bento check .` or `bento check --comparison=head .` taking 8 or 16 seconds.

**Comment by pabloest** [comment]
Perhaps related: it seems some Bento tools run twice (or at least appear to):
![Screen Recording 2020-01-22 at 1 51 34 PM 2020-01-22 14_28_37](https://user-images.githubusercontent.com/302941/72940353-97e8fd80-3d23-11ea-83ec-3bca7c9cd266.gif)

**Comment by dlukeomalley** [comment]
Related ticket (re: confusion of what is happening): #233

**Comment by dlukeomalley** [comment]
Fixed in 0.8.0b10
